### DISCUSSION TITLE
使用代理出现异常

### DISCUSSION THREAD

**Discussion by Toserk1024** [AlistGo/alist#4962] [Q&A]
Alist版本：3.25.1
使用代理下载文件，返回了{"code": 500,"msg": "unexpected end of JSON input"}前几个版本还是正常的。
有人知道是怎么回事吗？

**Comment by aozaki-kuro** [comment]
同样，我这里使用代理也出现了错误，前两天按照官网教程设置是 ok 的

**Comment by anwen-anyi** [comment]
https://github.com/alist-org/alist/issues/4971 已经修复，方便的话可以帮忙测试一下有没有问题

**Reply by aozaki-kuro** [reply]
docker 版本（ https://github.com/alist-org/alist/commit/df6b306fcee906c9b5877f576f6941d605c7c3db ）测试 ok，期待尽快 release 🥺

**Reply by anwen-anyi** [reply]
> docker 版本（ [df6b306](https://github.com/alist-org/alist/commit/df6b306fcee906c9b5877f576f6941d605c7c3db) ）测试 ok，期待尽快 release pleading_face

感谢测试，你现在先用着测试版吧~ 等发版在更新就可以~ 么么哒
